boards/posix/nrf52_bsim/nrf52_bsim.yaml

```diff
@@ -2,10 +2,12 @@ identifier: nrf52_bsim
 name: NRF52 BabbleSim board
 type: native
 arch: posix
+simulation: bsim
 env:
   - BSIM_OUT_PATH
 toolchain:
   - zephyr
 testing:
-  ignore_tags:
-    - drivers
+  default: true
+  only_tags:
+    - bluetooth
```

Comment:
Not sure about this change. I'd like this to run in CI on the bluetooth tag, but IDK if this is the way to do it.